### ISSUE TITLE
issue about build example code

### ISSUE THREAD

**Issue by dwgeneral** [smallnest/gen#115]
I was successfully run `$ gen --sqltype=sqlite3 \ .....`

now I would build example code  `$ cd ./example && make example `,but I got this error: 

```
build example server
swag init --dir .  --generalInfo ./app/server/main.go
2020/07/07 17:21:51 Generate swagger docs....
2020/07/07 17:21:51 Generate general API Info, search dir:.
2020/07/07 17:21:51 ParseComment error in file apis/artists.go :can not find schema type: "apis.PagedResults"
make: *** [example] Error 1
```

I checked the app/server/main.go, and I found these packages cound not be imported
```
"github.com/alexj212/generated/apis"
"github.com/alexj212/generated/daos"
_ "github.com/alexj212/generated/docs"
"github.com/alexj212/generated/models"
```

some suggestions ?

**Comment by alexj212** [comment]
Can you post your full command line - It looks like go.mod was not generated 

type PagedResults struct exists in example/apis/router.go

**Comment by dwgeneral** [comment]
I fixed it. the reason is `template/router.go.tmpl` throw some mistakes. 

`tmp.TableInfo, _ = {{.modelPackageName}}.GetTableInfo("{{$tableName}}")`  

should change to 

`tmp.TableInfo, _ = {{$.modelPackageName}}.GetTableInfo("{{$tableName}}")`

**Comment by alexj212** [comment]
Fixed in v0.9.20
